### PR TITLE
Onboard the official build to 1ES pipeline templates

### PR DESCRIPTION
**Pull request by maryamariyan**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -18,21 +18,36 @@ schedules:
...
+  - template: /eng/common/templates-official/variables/pool-providers.yml@self
   - name: _TeamName
     value: Roslyn
+resources:
+    repositories:
+    - repository: 1ESPipelineTemplates
+      type: git
+      name: 1ESPipelineTemplates/1ESPipelineTemplates
+      ref: refs/tags/release
+      extends:
```
Comment (jjonescz) [inline azure-pipelines.yml:30]: This is wrongly indented, should be at column 0

File: azure-pipelines.yml [hunk 2]
```diff
@@ -1,4 +1,6 @@
-# Branches that trigger builds on PR
+# This Yaml Document has been converted by ESAI Yaml Pipeline Conversion Tool.
+# Please make sure to check all the converted content, it is your team's responsibility to make sure that the pipeline is still valid and functions as expected.
+# This pipeline will be extended to the MicroBuild template
```
Comment (tmat) [inline azure-pipelines.yml:3]: Let's remove this comment. It's not useful.

File: azure-pipelines.yml [hunk 3]
```diff
@@ -8,32 +10,32 @@ pr:
...
 variables:
 - template: /eng/common/templates-official/variables/pool-providers.yml@self
 - name: _TeamName
   value: Roslyn
+- name: TeamName
+  value: Roslyn
 resources:
   repositories:
-  - repository: 1ESPipelineTemplates
+  - repository: MicroBuildTemplate
```
Comment (tmat) [inline azure-pipelines.yml:27]: Where is this repository?

Comment (mmitche) [reply azure-pipelines.yml:27]: There is a sibling project in dnceng which contains the repo.

**Review by jjonescz — APPROVED** [review]

**Pull request merged**